src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -31,6 +31,9 @@
       <Link>Common\System\Numerics\Hashing\HashHelpers.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(TargetGroup)' != 'portable_net40+sl4+win8+wp8'">
```

Comment:
I would either add a comment here or in the commit message (or both) saying why you didn't add it for the portable profile.